packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -421,11 +517,9 @@ export class VSXExtensionEditorComponent extends AbstractVSXExtensionComponent {
         </React.Fragment>;
     }
 
-    protected body: HTMLElement | undefined;
-
     // TODO replace with webview
     readonly openLink = (event: React.MouseEvent) => {
-        if (!this.body) {
+        if (!this.props.extension.body) {
```

Comment:
body ref should belong to this component, not to its parent, please don't break encapsulation